pkg/setting/provider.go

```diff
@@ -90,6 +93,7 @@ type ReloadHandler interface {
 }
 
 type SettingsBag map[string]map[string]string
+type VerboseSettingsBag map[string]map[string]map[string]string
```

Comment:
```suggestion
type VerboseSettingsBag map[string]SettingsBag
```

I think you can do this

Reply:
Now that I see the source is at the end this might not make sense and only confuse